apps/src/code-studio/clientState.js

```diff
@@ -172,7 +172,7 @@ clientState.allLevelsProgress = function () {
  * @returns {number}
  */
 clientState.lines = function () {
-  var linesStr = $.cookie('lines');
+  var linesStr = cookies.get('lines');
```

Comment:
could be const, tho i understand leaving things how they are so that this change is minimal